libpod/healthcheck.go

```diff
@@ -156,7 +148,18 @@ func (c *Container) runHealthCheck(ctx context.Context, isStartup bool) (define.
 	hcl := newHealthCheckLog(timeStart, timeEnd, returnCode, eventLog)
 	logStatus, err := c.updateHealthCheckLog(hcl, inStartPeriod, isStartup)
 	if err != nil {
-		return hcResult, "", fmt.Errorf("unable to update health check log %s for %s: %w", c.healthCheckLogPath(), c.ID(), err)
+		path, err := c.healthCheckLogPath()
```

Comment:
So, from reading this - do we support logging to file and journald *at the same time*? That seems unnecessary - one or the other should be sufficient.